cli/src/commands/application/init.rs

```diff
@@ -179,6 +179,18 @@ async fn configure_namespace(
     wk_client: &mut WKClient,
     appsignal_apps: &mut Option<Vec<AppsignalAppsQueryAppsignalApps>>,
 ) -> Result<ApplicationNamespaceConfig, WKCliError> {
+    let application_name = inquire::Text::new("Pipeline application name")
```

Comment:
May I know what does the pipeline application name do? because there is no explanation during the prompt.

Reply:
like does wukong use the name to provisioning new pipeline? or use it to link the existing pipeline? What should I put in this field?

Reply:
confused when seeing this 

<img width="730" alt="Screenshot 2024-05-29 at 5 22 14 PM" src="https://github.com/mindvalley/wukong-cli/assets/7545747/fb4efb8c-1296-4eac-8f67-7a7e0c1b4bbd">